### ISSUE TITLE
add ParentNode['replaceChildren'] to dom.generated.d.ts

### ISSUE THREAD

**Issue by btoo** [microsoft/TypeScript#43456]
# lib Update Request

## Configuration Check

My compilation *target* is `ES2021` and my *lib* is `lib.dom.d.ts`.

## Missing / Incorrect Definition

`ParentNode.replaceChildren()`

## Sample Code

```typescript
document.body.replaceChildren();
document.body.replaceChildren('hello world', document.createElement('button'));
```

## Documentation Link

[MDN](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/replaceChildren)
[WHATWG](https://dom.spec.whatwg.org/#dom-parentnode-replacechildren)

**Comment by Ragzouken** [comment]
relevant issue from the repository the types are generated from: https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/985

**Comment by kidlj** [comment]
Has this been merged into upstream? I have `n.parentNode.replaceChildren` successfully executed but TypeScript reports error:

```
TS2339: Property 'replaceChildren' does not exist on type 'ParentNode'.
```

for the following code:
```typescript
function mount(n: Node) {
	for (let p = n.parentNode; p != null && p.nodeName != "HTML"; p = p.parentNode) {
		(<ParentNode>p).replaceChildren(n)
		n = p
	}
}
```

Also if I don't force casting `(<ParentNode>p)`, it reports:
```
TS2339: Property 'replaceChildren' does not exist on type 'Node & ParentNode'.
```

Don't know why.

**Comment by kidlj** [comment]
Well, don't compile in production mode. I have to use type casting `(<any>p).replaceChildren(n)` for it to build.

**Comment by jakebailey** [comment]
This was fixed back in #44684 for TS 4.4+. The examples in this thread all compile. [Playground Link](https://www.typescriptlang.org/play?ts=4.4.4#code/GYVwdgxgLglg9mABAWzuKAKMAuRA5OAEwFMBKRAbwFgAoASGDgCdEMAbYqRAB0QF5EYAHTcAhk2JgoBEgG4eiAIQCwINm0QAyTTyFgixPKOTElAgEQAJACoBZADLn5vAdxHjJ0g+Wr06biW42UQhiAGEACxg2QgkwLFJaOjokVySAX1p0oA)
